### ISSUE TITLE
player.stv.tv

### ISSUE THREAD

**Issue by suFDHAI78fyw78q09Y98yfd** [ytdl-org/youtube-dl#20586]
---
- [x ] I've **verified** and **I assure** that I'm running youtube-dl **2019.04.01**

### Before submitting an *issue* make sure you have:
- [ x] At least skimmed through the [README](https://github.com/ytdl-org/youtube-dl/blob/master/README.md), **most notably** the [FAQ](https://github.com/ytdl-org/youtube-dl#faq) and [BUGS](https://github.com/ytdl-org/youtube-dl#bugs) sections
- [x ] [Searched](https://github.com/ytdl-org/youtube-dl/search?type=Issues) the bugtracker for similar issues including closed ones
- [ ] Checked that provided video/audio/playlist URLs (if any) are alive and playable in a browser.

### What is the purpose of your *issue*?
- [x] Bug report (encountered problems with youtube-dl)
- [x] Site support request (request for adding support for a new site)

---

```
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['--geo-bypass-country', 'GB', '--ignore-config', '-v
', '-F', 'https://player.stv.tv/episode/3ikk/liberty-american-cookbook/']
[debug] Encodings: locale cp1252, fs mbcs, out cp850, pref cp1252
[debug] youtube-dl version 2019.04.01
[debug] Python version 3.4.4 (CPython) - Windows-XP-5.1.2600-SP3
[debug] exe versions: ffmpeg N-91691-g962c931-Reino, ffprobe N-91691-g962c931-Re
ino, rtmpdump 2.4
[debug] Proxy map: {}
[debug] Using fake IP 25.66.14.113 (GB) as X-Forwarded-For.
[generic] liberty-american-cookbook: Requesting header
WARNING: Falling back on generic information extractor.
[generic] liberty-american-cookbook: Downloading webpage
[generic] liberty-american-cookbook: Extracting information
ERROR: Unsupported URL: https://player.stv.tv/episode/3ikk/liberty-american-cook
book/
Traceback (most recent call last):
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmp48z7ci
11\build\youtube_dl\YoutubeDL.py", line 794, in extract_info
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmp48z7ci
11\build\youtube_dl\extractor\common.py", line 529, in extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmp48z7ci
11\build\youtube_dl\extractor\generic.py", line 3320, in _real_extract
youtube_dl.utils.UnsupportedError: Unsupported URL: https://player.stv.tv/episod
e/3ikk/liberty-american-cookbook/
...
<end of log>
```

---

### If the purpose of this *issue* is a *site support request* please provide all kinds of example URLs support for which should be included (replace following example URLs by **yours**):

- Single video: https://player.stv.tv/episode/3ikk/liberty-american-cookbook/
- Single video: https://player.stv.tv/episode/3jgl/liberty-american-cookbook/

---
Support for.
player.stv.tv
Thanks.

**Comment by suFDHAI78fyw78q09Y98yfd** [comment]
That I knew this would be a probability and the reason why I tried it with the following command --geo-bypass-country "GB" to ensure any geo restrictions are passed.

I checked again now for 'Country Codes ISO 3166-2' the code "GB" does actually cover all 3 countries and 1 province of UK. Of which one of the countries is Scotland where STV is located.

If you or another youtube-dl maintainer can add support for this site it would be appreciated.
